test/erlcloud_sns_tests.erl

```diff
@@ -121,7 +122,11 @@ validate_params(Body, Expected) ->
 %% Validates the query body and responds with the provided response.
 -spec input_expect(string(), [expected_param()]) -> fun().
 input_expect(Response, Expected) ->
-    fun(_Url, post, _Headers, Body, _Timeout, _Config) ->
+    fun
+        (_Url, post, _Headers, "test message" = Body, _Timeout, _Config) ->
+            validate_params(Body, Expected),
+            {ok, {{200, "OK"}, [], <<"">>}};
```

Comment:
```suggestion
            {ok, {{200, "OK"}, [], <<>>}};
```

Reply:
Removed this clause as it was not being used